src/RwaConduit.sol

```diff
@@ -13,7 +18,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-pragma solidity 0.5.12;
+pragma solidity ~0.6.12;
```

Comment:
note: introducing the `~` modifier should be equivalent 0.6.12 <= n < 0.7.0.
as discussed in chat, I don't see any benefit in using it, sharing here as well for general discussion.

Reply:
I would remove it if not necessary